config.py

```diff
@@ -10,6 +10,7 @@ class Config(object):
 class DevConfig(Config):
     DEBUG = True
     DATABASE_URI = getenv('DEV_DATABASE_URI', '')
+    SQLALCHEMY_DATABASE_URI = 'postgresql://psql_user:Ab132xw@127.0.0.1:5432/psql_db'
```

Comment:
Put it into environment variable? or just new line like this would be fine:
`SQLALCHEMY_DATABASE_URI = \
        'postgresql://psql_user:Ab132xw@127.0.0.1:5432/psql_db'`

Reply:
I think 5 env variables will be best:
-DB_ USER
-DB_NAME
-DB_HOST
-DB_PASSWORD
-DB_PORT

and later use format to built this URI:
DB_USER = getenv('DB_USER', '<some_default_value>')
....
'postgresql://{0}:{1} ....'.format(DB_USER, ..)

And don't forget to set it in docker-compose :)